### DISCUSSION TITLE
How to add a different partition to search index?

### DISCUSSION THREAD

**Discussion by meier-johannes94** [junegunn/fzf#2472] [Q&A] [answered]
Hello Dear Developers, 

I have most of my data saved in the direcory dev/nvme0n1p3 (different partition). 
If I search for files in that directory he doesn't find it. 
Is it possible to add those files to the index? 

Thanks a lot!

Johannes

**Comment by junegunn** [comment, marked as answer]
fzf is not a `find` alternative. It's an interactive text filter rather than a file system explorer. Please read through the README page to learn how to properly use fzf.
